src/ducks/contents.js

```diff
@@ -26,12 +26,12 @@ const projectContentsReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchProjectContents = () => {
+const fetchProjectContents = (project_id) => {
```

Comment:
My bad, this should actually be requesting a list of all `ProjectContents` resources for a given project, then listing them by language.